functions/send_time_off_request_to_manager/logger.ts

```diff
@@ -0,0 +1,21 @@
+import * as log from "https://deno.land/std@0.157.0/log/mod.ts";
+
+export const Logger = function (
```

Comment:
Once we add this module to deno-slack-sdk, we can delete this and switch to the one provided by the SDK